<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qplot</class>
 <widget class="QMainWindow" name="qplot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qplot</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/icons/appicon.svg</normaloff>:/icons/icons/appicon.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QFrame" name="r_frame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>50</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QCustomPlot" name="plotWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="l_fram">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QWidget" name="main_wdget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QListWidget" name="lv_functions_view"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="lbl_func">
               <property name="text">
                <string>y = </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="le_func"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Color (RGB): </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="sb_red">
               <property name="maximum">
                <number>255</number>
               </property>
               <property name="singleStep">
                <number>1</number>
               </property>
               <property name="value">
                <number>100</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="sb_green">
               <property name="maximum">
                <number>255</number>
               </property>
               <property name="value">
                <number>100</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="sb_blue">
               <property name="maximum">
                <number>255</number>
               </property>
               <property name="value">
                <number>255</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="settings_widget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QLabel" name="lbl_xBegin">
                 <property name="text">
                  <string>X Begin:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="lbl_xEnd">
                 <property name="text">
                  <string>X End:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="lbl_step">
                 <property name="text">
                  <string>Step:</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QDoubleSpinBox" name="sb_xBegin">
                 <property name="minimum">
                  <double>-10000000.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>10000000.000000000000000</double>
                 </property>
                 <property name="value">
                  <double>-10000.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="sb_xEnd">
                 <property name="maximum">
                  <double>10000000.000000000000000</double>
                 </property>
                 <property name="value">
                  <double>10000.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="sb_step">
                 <property name="decimals">
                  <number>5</number>
                 </property>
                 <property name="minimum">
                  <double>0.000000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>5.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.010000000000000</double>
                 </property>
                 <property name="value">
                  <double>0.010000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="btn_approw">
             <property name="text">
              <string>Approw</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="btn_add_func">
            <property name="text">
             <string>Add Function</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_setting">
            <property name="text">
             <string>Settings</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>686</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProject">
    <property name="title">
     <string>Project</string>
    </property>
    <addaction name="actionCreate"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="menuProject"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCreate">
   <property name="text">
    <string>Create</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_setting</sender>
   <signal>toggled(bool)</signal>
   <receiver>settings_widget</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>637</x>
     <y>545</y>
    </hint>
    <hint type="destinationlabel">
     <x>491</x>
     <y>458</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_setting</sender>
   <signal>toggled(bool)</signal>
   <receiver>settings_widget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>637</x>
     <y>545</y>
    </hint>
    <hint type="destinationlabel">
     <x>491</x>
     <y>458</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
